make-debs-arm32.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
also, i'd suggest drop this change for the same reason. i think we need to provide the machinery in the `CMakeLists.txt` and source code to improve the portablity of ceph. and to materialise some of the typical settings as `make-*` scripts under the root of source directory. but i don't think we need to keep all of them there.

Reply:
Also do not directly call `bash` in `/bin`
but make it liek:
`#!/usr/bin/env bash`
Which will make it work on platforms where there is no `bash` in `/bin`